patches/chrome-browser-ui-webui-password_manager-password_manager_ui.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/webui/password_manager/password_manager_ui.cc b/chrome/browser/ui/webui/password_manager/password_manager_ui.cc
+index 4357a2c3f8c9ea7e2e2c75705c6a3865f4e9ef03..ccadff8d11db2c7d4fe33b6190c3a0e36945b06a 100644
+--- a/chrome/browser/ui/webui/password_manager/password_manager_ui.cc
++++ b/chrome/browser/ui/webui/password_manager/password_manager_ui.cc
+@@ -486,6 +486,7 @@ content::WebUIDataSource* CreateAndAddPasswordsUIHTMLSource(
+                    profile, chrome::FaviconUrlFormat::kFavicon2));
+ 
+   webui::SetupChromeRefresh2023(source);
```

Comment:
define `SetupChromeRefresh2023(SOURCE) SetupChromeRefresh2023(SOURCE); .....`

Reply:
Fixed in https://github.com/brave/brave-core/pull/19398/commits/dc77a78e2f88684897961a74f0f5fcdb143304fb